### DISCUSSION TITLE
Welcome to pnc-repressurized Discussions!

### DISCUSSION THREAD

**Discussion by desht** [TeamPneumatic/pnc-repressurized#678] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jebjordan** [comment]
What is the API for cc?

It's described as the same as previously in the pneumaticcraft,
but It's been so damn long and I don't even think I used it back then.
Is it possible to have a good ol' documentation page on the wiki for all the exposed lua api functions and their args and such?

Anyways

**Reply by desht** [reply]
Some wiki documentation would be good, but I don't have the time to build a complete offline set of docs, sorry.

Anyway, CC integration is fairly well described in the in-game manual (Patchouli required):
* General overview in the "Computer Integration" page in the "Base Concepts" section including Lua methods common to many machines (pressure & heat querying, mainly)
* Machine-specific Lua methods in that machine's page (e.g. Elevator Base, Universal Sensor...)
* Drone Interface has many many methods all documented in its page
* Computer Control programming widget allows drones to connect to a Drone Interface

**Reply by jebjordan** [reply]
Thank you.
I did not realize there was some alright in-game docs on the api.
Thanks for maintaining and reviving this wonderful mod by the way. :)

by the way, while I have you...
"Filled" is supposed to work as an area type for addArea right?

It does not appear to be working. hmm

**Reply by desht** [reply]
Please create an issue for that (and include your Lua code etc. and any instructions necessary to reproduce)
